plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -43,11 +45,26 @@ void AudioFileProcessorWaveView::updateSampleRange()
 	if (m_sample->sampleSize() > 1)
 	{
 		const f_cnt_t marging = (m_sample->endFrame() - m_sample->startFrame()) * 0.1;
-		m_from = qMax(0, m_sample->startFrame() - marging);
-		m_to = qMin<size_t>(m_sample->endFrame() + marging, m_sample->sampleSize());
+		setFrom(qMax(0, m_sample->startFrame() - marging));
+		setTo(qMin<size_t>(m_sample->endFrame() + marging, m_sample->sampleSize()));
```

Comment:
```suggestion
		setFrom(m_sample->startFrame() - marging);
		setTo(m_sample->endFrame() + marging);
```
Using the new methods as intended

Reply:
Good catch! Fixed with commit 26d7c48180d.